packages/grafana-ui/src/components/Alert/Alert.tsx

```diff
@@ -55,42 +56,55 @@ export const Alert = React.forwardRef<HTMLDivElement, Props>(
     return (
       <div
         ref={ref}
-        className={cx(styles.alert, className)}
+        className={cx(styles.wrapper, className)}
         data-testid={selectors.components.Alert.alertV2(severity)}
         role={role}
         aria-label={ariaLabel}
         {...restProps}
       >
-        <div className={styles.icon}>
-          <Icon size="xl" name={getIconFromSeverity(severity)} />
-        </div>
+        <Box
+          display="flex"
+          backgroundColor={severity}
+          borderRadius="default"
+          paddingY={1}
+          paddingX={2}
+          borderStyle="solid"
+          borderColor={severity}
+          alignItems="stretch"
+          boxShadow={elevated ? 'z3' : undefined}
+        >
+          <Box paddingTop={1} paddingRight={2}>
+            <div className={styles.icon}>
+              <Icon size="xl" name={getIconFromSeverity(severity)} />
+            </div>
```

Comment:
Not really the point of this PR, but Icon should just have a color prop.

Reply:
Yeah... I thought of wrapping this in `Text` but that feels icky as well, I don't know why `Icon` doesn't have a `color` props 🤷

Reply:
Not so sure about the color prop, people might get a little bit too excited with it.

Reply:
Keeping in mind it would be constrained to the same tokened colours.